diff --git "a/D:\\Temp\\TortoiseGit\\secF905.tmp\\security_settings_encrypted-1ca28f4-left.xml" "b/D:\\Temp\\TortoiseGit\\secF904.tmp\\security_settings_encrypted-27e876f-right.xml"
index e82dc56..7a165de 100644
--- "a/D:\\Temp\\TortoiseGit\\secF905.tmp\\security_settings_encrypted-1ca28f4-left.xml"
+++ "b/D:\\Temp\\TortoiseGit\\secF904.tmp\\security_settings_encrypted-27e876f-right.xml"
@@ -26,6 +26,16 @@
             android:summary="@string/crypt_keeper_encrypted_summary"
         />
 
+        <Preference
+            android:key="change_crypt_password"
+            android:title="@string/change_crypt_password_title"
+            android:summary="@string/change_crypt_password_summary">
+
+            <intent android:action="android.intent.action.MAIN"
+            android:targetPackage="com.android.settings"
+            android:targetClass="com.android.settings.ChooseEncryptionPassword" />
+        </Preference>
+
     </PreferenceCategory>
 
 
