diff --git "a/D:\\Temp\\TortoiseGit\\Qui5C97.tmp\\QuickSettingsTiles-51e3d76-left.java" "b/D:\\Temp\\TortoiseGit\\Qui5C86.tmp\\QuickSettingsTiles-51264a9-right.java"
index 552fc6e..8f3c0fc 100644
--- "a/D:\\Temp\\TortoiseGit\\Qui5C97.tmp\\QuickSettingsTiles-51e3d76-left.java"
+++ "b/D:\\Temp\\TortoiseGit\\Qui5C86.tmp\\QuickSettingsTiles-51264a9-right.java"
@@ -33,7 +33,6 @@ import android.content.SharedPreferences;
 import android.graphics.drawable.Drawable;
 import android.os.Bundle;
 import android.os.UserHandle;
-import android.os.Vibrator;
 import android.provider.Settings;
 import android.text.TextUtils;
 import android.util.TypedValue;
@@ -90,8 +89,6 @@ public class QuickSettingsTiles extends Fragment {
     private int mTileTextSize;
     private int mTileTextPadding;
 
-    private boolean mHasVibrator;
-
     @Override
     public View onCreateView(LayoutInflater inflater,
             ViewGroup container, Bundle savedInstanceState) {
@@ -139,9 +136,6 @@ public class QuickSettingsTiles extends Fragment {
         }
         mTileAdapter = new TileAdapter(getActivity());
 
-        Vibrator vibrator = (Vibrator) getActivity().getSystemService(Context.VIBRATOR_SERVICE);
-        mHasVibrator = vibrator.hasVibrator();
-
         return mDragView;
     }
 
@@ -230,7 +224,7 @@ public class QuickSettingsTiles extends Fragment {
                         QSConstants.TILE_SCREENTIMEOUT).getTitleResId()
                 || titleId == QuickSettingsUtil.TILES.get(
                             QSConstants.TILE_RINGER).getTitleResId()
-                        && mHasVibrator
+                        && DeviceUtils.deviceSupportsVibrator(getActivity())
                 || titleId == QuickSettingsUtil.TILES.get(
                             QSConstants.TILE_MUSIC).getTitleResId()
                 || QuickSettingsUtil.isTileAvailable(QSConstants.TILE_NETWORKMODE)
@@ -274,7 +268,6 @@ public class QuickSettingsTiles extends Fragment {
                 }
             }
         });
-        final boolean hasVibrator = mHasVibrator;
         mDragView.setOnItemClickListener(new OnItemClickListener() {
             @Override
             public void onItemClick(AdapterView<?> arg0, View arg1, int arg2, long arg3) {
@@ -291,7 +284,7 @@ public class QuickSettingsTiles extends Fragment {
                         showDialogInner(DLG_NETWORK_MODE);
                     }
                     if (tiles.get(arg2).equals(QSConstants.TILE_RINGER)
-                            && hasVibrator) {
+                            && DeviceUtils.deviceSupportsVibrator(getActivity())) {
                         showDialogInner(DLG_RINGER);
                     }
                     if (tiles.get(arg2).equals(QSConstants.TILE_MUSIC)) {
