diff --git "a/D:\\Temp\\TortoiseGit\\cr_A500.tmp\\cr_strings-1ca28f4-left.xml" "b/D:\\Temp\\TortoiseGit\\cr_A4F0.tmp\\cr_strings-27e876f-right.xml"
index 77b0fe8..9ab68c2 100644
--- "a/D:\\Temp\\TortoiseGit\\cr_A500.tmp\\cr_strings-1ca28f4-left.xml"
+++ "b/D:\\Temp\\TortoiseGit\\cr_A4F0.tmp\\cr_strings-27e876f-right.xml"
@@ -135,6 +135,7 @@
     <string name="reset_message">Alle Einträge zurücksetzen?</string>
     <string name="style_title">Aussehen</string>
     <string name="shortcut_action_airplane">Flugzeugmodus</string>
+    <string name="shortcut_action_expanded_desktop">Erweiterter Desktop</string>
     <string name="shortcut_action_power_off">Ausschalten</string>
     <string name="shortcut_action_reboot">Neustart</string>
     <string name="shortcut_action_sound">Sound</string>
@@ -165,7 +166,7 @@
     <string name="shortcut_action_max">Maximale Größe der Einträge erreicht.</string>
     <string name="shortcut_action_warning">Warnung</string>
     <string name="shortcut_action_warning_message">Es ist nicht möglich den letzen Eintrag zu löschen</string>
-    <string name="shortcut_action_disable_message">Ausgeschaltet.Füge einen Eintrag hinzu zum aktivieren dieses Features</string>
+    <string name="shortcut_action_disable_message">Ausgeschaltet. Füge einen Eintrag hinzu zum aktivieren dieses Features.</string>
     <string name="shortcut_image_not_valid">Das gewählte Programm verfügt über kein erlaubtes Icon oder unterstützt keine Schneidefunktion. Benutze ein anderes Programm.</string>
     <string name="shortcut_duplicate_entry">Die gewählte Aktion kann nicht zweimal hinzugefügt werden.</string>
 
@@ -408,7 +409,7 @@
     <!-- Lock Screen Targets -->
     <string name="lockscreen_target_info">Ziehe den Slider auf die Ziele um Apps zuzuweisen.</string>
     <string name="lockscreen_target_title">Sliderziele</string>
-    <string name="lockscreen_target_summary">Anzeigen und ändern von eigenen Sperrbildschirmzielen</string>
+    <string name="lockscreen_target_summary">Anzeigen und ändern von eigenen Sperrbildschirmzielen.</string>
     <string name="lockscreen_target_color_title">Farbe Icon Ziel</string>
     <string name="lockscreen_eight_targets_title">Acht Sliderziele</string>
     <string name="lockscreen_eight_targets_summary">Aktiviert acht Sliderziele.</string>
@@ -459,7 +460,7 @@
     <string name="halo_reversed_title">HALO umkehren</string>
     <string name="halo_reversed_summary">Gilt für Task-Umschaltung und Bewegungs-Gesten.</string>
     <string name="halo_state_title">HALO Regeln</string>
-    <string name="halo_state_summary">Aktiviere Black/White-Listen Benachrichtigungen</string>
+    <string name="halo_state_summary">Aktiviere Black/White-Listen Benachrichtigungen.</string>
     <string name="halo_state_black">Black-Liste</string>
     <string name="halo_state_white">White-Liste</string>
     <string name="halo_pause_title">Pausiere aktive APP</string>
@@ -575,7 +576,7 @@
     <string name="ad_excluded_apps_title">Auszuschließende Apps</string>
     <string name="ad_excluded_apps_summary">Auszuschließende Apps werden nicht im Active Display erscheinen.</string>
     <string name="ad_brightness_title">Helligkeit Benachrichtigungen</string>
-    <string name="ad_brightness_summary">Helligkeitslevel der angezeigten Benachrichtigungen.\n Wenn automatische Helligkeit nicht aktiviert ist.</string>
+    <string name="ad_brightness_summary">Helligkeitslevel der angezeigten Benachrichtigungen.\nWenn automatische Helligkeit nicht aktiviert ist.</string>
     <string name="ad_show_date_title">Zeige Datum</string>
     <string name="ad_show_date_summary">Zeigt das Datum über der Uhr an.</string>
     <string name="ad_show_ampm_title">Zeige AM/PM</string>
@@ -640,7 +641,7 @@
     <string name="notif_alpha_title">Transparenz Benachrichtigungszeile</string>
     
     <!-- Custom notification panel header -->
-    <string name="omni_features_title">Omni Features</string>
+    <string name="omni_features_title">Omni Eigenschaften</string>
     <string name="notification_drawer_category_title">Benachrichtigungsfeld</string>
     <string name="custom_statusbar_header_title">Bild Benachrichtigungsfeld</string>
     <string name="custom_statusbar_header_summary">Ersetzt den Hintergrund des Benachrichtigungsfeld-Kopfes durch ein Hintergrundbild mit dem aktuellen Datum und der Uhrzeit.</string>
@@ -675,19 +676,28 @@
     <string name="title_tile_autorotate">Ausrichtung</string>
     <string name="title_tile_settings">Einstellungen</string>
     <string name="title_tile_sound">Sound</string>
-    <string name="title_tile_sync">Sync</string>
+    <string name="title_tile_sync">Synchronisieren</string>
     <string name="title_tile_wifiap">Wi-Fi AP</string>
     <string name="title_tile_wifi">Wi-Fi</string>
-    <string name="title_tile_user">User wechslen</string>
+    <string name="title_tile_user">Benutzer wechseln</string>
     <string name="title_tile_networkmode">Netzwerk-Modus</string>
     <string name="title_tile_nfc">NFC</string>
     <string name="title_tile_screen_timeout">Bildschirm-Timeout</string>
+    <string name="title_tile_quick_record">Schnellaufnahme</string>
     <string name="title_tile_quiet_hours">Ruhige Stunden</string>
     <string name="title_tile_volume">Lautstärke</string>
     <string name="title_tile_expanded_desktop">Erweiterter Desktop</string>
     <string name="title_tile_music">Musik</string>
     <string name="title_tile_reboot">Neustart</string>
+    <string name="dynamic_tile_ime_switcher">IME Wechsel</string>
+    <string name="dynamic_tile_usb_tether">USB Tether</string>
+    <string name="dynamic_tile_alarm">Wecker</string>
+    <string name="dynamic_tile_bugreport">Bugreport</string>
+    <string name="dynamic_tiles_title">Dynamische Kacheln</string>
     <string name="tiles_reset_message">Standardmäßig angezeigte Kacheln und Layout wiederherstellen?</string>
+    <string name="disable_qs">Schnelleinstellungen sind deaktiviert.</string>
+    <string name="disable_qs_message">Alle Kacheln wurden entfernt. Schnelleinstellungen sind deaktiviert.\n\nUm Kacheln hinzufügen zu aktivieren.</string>
+    <string name="help_qs_message">Um eine Schnelleinstellungs-Kachel hinzuzufügen, die Hinzufügen-Taste drücken und die gewünschte Kachel auswählen. Um die Kacheln neu anzuordnen die gewünschte Kachel gedrückt halten und in die gewünschte Position ziehen. Um eine Kachel zu löschen die gewünschte Kachel auf die Lösch-Taste ziehen, welcher während dem verschieben der Kacheln erscheint.\n\nUm eine Dynamische Kachel zu erstellen das jeweilige Icon oben rechts auf dem Bildschirm drücken.\n\nUm die Schnelleinstellungen zu deaktivieren, alle Kacheln löschen.</string>
 
     <!-- QuickSettings style options -->
     <string name="quicksettings_tiles_style">Aussehen</string>
@@ -758,6 +768,20 @@
 
     <!-- Advanced, Device specific (DeviceParts) settings -->
     <string name="advanced_settings_title">Erweitert</string>
+    
+    <!-- Chamber of Secrets -->
+    <string name="chamber_title">Geheimkammer</string>
+    <string name="chamber_toast">Die geheime Kammer wurde geöffnet.</string>
+
+     <!-- Title for PreferenceScreen to change encryption password [CHAR LIMIT=22] -->
+    <string name="change_crypt_password_title">Passwort ändern</string>
+    <!-- Summary for PreferenceScreen to change encryption password [CHAR LIMIT=45] -->
+    <string name="change_crypt_password_summary">Passwort ändern mit dem das Gerät verschlüsselt wird.</string>
+
+    <!-- Header on first screen of choose encryption password [CHAR LIMIT=30] -->
+    <string name="encryptionpassword_choose_your_password_header">Setze Verschlüsslungs-Passwort</string>
+
+
 
 
 
