diff --git "a/D:\\Temp\\TortoiseGit\\dev29A7.tmp\\device_info_settings-51e3d76-left.xml" "b/D:\\Temp\\TortoiseGit\\dev29A6.tmp\\device_info_settings-51264a9-right.xml"
index 4764069..7d0b628 100644
--- "a/D:\\Temp\\TortoiseGit\\dev29A7.tmp\\device_info_settings-51e3d76-left.xml"
+++ "b/D:\\Temp\\TortoiseGit\\dev29A6.tmp\\device_info_settings-51264a9-right.xml"
@@ -87,6 +87,13 @@
                 <intent android:action="android.settings.LICENSE" />
             </PreferenceScreen>
 
+            <!-- CM License information -->
+            <PreferenceScreen
+                    android:key="cmlicense"
+                    android:title="@string/cmlicense_title">
+                <intent android:action="android.settings.CMLICENSE" />
+            </PreferenceScreen>
+
             <!-- Terms and conditions -->
             <PreferenceScreen
                     android:key="terms"
