diff --git "a/D:\\Temp\\TortoiseGit\\cr_B311.tmp\\cr_strings-1ca28f4-left.xml" "b/D:\\Temp\\TortoiseGit\\cr_B300.tmp\\cr_strings-27e876f-right.xml"
index cfa2f7a..8280263 100644
--- "a/D:\\Temp\\TortoiseGit\\cr_B311.tmp\\cr_strings-1ca28f4-left.xml"
+++ "b/D:\\Temp\\TortoiseGit\\cr_B300.tmp\\cr_strings-27e876f-right.xml"
@@ -5,7 +5,7 @@
     <!-- About phone screen, temasek version -->
     <string name="temasek_version_title">Temasek unofficial version</string>
     <!-- Do NOT Translate this -->
-    <string name="temasek_version_default">V24</string>
+    <string name="temasek_version_default">V25</string>
 
     <!-- General strings -->
     <string name="reset">Reset</string>
@@ -762,4 +762,12 @@
     <string name="chamber_title">Chamber of secrets</string>
     <string name="chamber_toast">The chamber of secrets has opened</string>
 
+     <!-- Title for PreferenceScreen to change encryption password [CHAR LIMIT=22] -->
+    <string name="change_crypt_password_title">Change password</string>
+    <!-- Summary for PreferenceScreen to change encryption password [CHAR LIMIT=45] -->
+    <string name="change_crypt_password_summary">Change the password used for device encryption</string>
+
+    <!-- Header on first screen of choose encryption password [CHAR LIMIT=30] -->
+    <string name="encryptionpassword_choose_your_password_header">Set encryption password</string>
+
 </resources>
