diff --git "a/D:\\Temp\\TortoiseGit\\NFC3130.tmp\\NFCProfileWriter-51e3d76-left.java" "b/D:\\Temp\\TortoiseGit\\NFC311F.tmp\\NFCProfileWriter-51264a9-right.java"
index 72b8c13..e858af4 100644
--- "a/D:\\Temp\\TortoiseGit\\NFC3130.tmp\\NFCProfileWriter-51e3d76-left.java"
+++ "b/D:\\Temp\\TortoiseGit\\NFC311F.tmp\\NFCProfileWriter-51264a9-right.java"
@@ -18,6 +18,7 @@ package com.android.settings.profiles;
 
 import java.util.UUID;
 
+import android.app.ActionBar;
 import android.app.Activity;
 import android.app.PendingIntent;
 import android.app.Profile;
@@ -29,6 +30,7 @@ import android.nfc.NfcAdapter;
 import android.nfc.Tag;
 import android.os.Bundle;
 import android.util.Log;
+import android.view.MenuItem;
 import android.widget.Toast;
 
 import com.android.settings.R;
@@ -61,6 +63,20 @@ public class NFCProfileWriter extends Activity {
 
         setContentView(R.layout.nfc_writer);
         setTitle(R.string.profile_write_nfc_tag);
+
+        ActionBar mActionBar = getActionBar();
+        if (mActionBar != null) {
+            mActionBar.setDisplayHomeAsUpEnabled(true);
+        }
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        if (item.getItemId() == android.R.id.home) {
+            finish();
+            return true;
+        }
+        return false;
     }
 
     @Override
