diff --git "a/D:\\Temp\\TortoiseGit\\Boo19A2.tmp\\BootReceiver-a306630-left.java" "b/D:\\Temp\\TortoiseGit\\Boo19A1.tmp\\BootReceiver-4528e68-right.java"
index 3ff8a4f..a7392c2 100644
--- "a/D:\\Temp\\TortoiseGit\\Boo19A2.tmp\\BootReceiver-a306630-left.java"
+++ "b/D:\\Temp\\TortoiseGit\\Boo19A1.tmp\\BootReceiver-4528e68-right.java"
@@ -30,6 +30,7 @@ import com.android.settings.Utils;
 import com.android.settings.hardware.DisplayColor;
 import com.android.settings.hardware.DisplayGamma;
 import com.android.settings.hardware.VibratorIntensity;
+import com.android.settings.location.LocationSettings;
 
 import java.util.Arrays;
 import java.util.List;
@@ -75,6 +76,7 @@ public class BootReceiver extends BroadcastReceiver {
         DisplayGamma.restore(ctx);
         VibratorIntensity.restore(ctx);
         DisplaySettings.restore(ctx);
+        LocationSettings.restore(ctx);
     }
 
     private void initFreqCapFiles(Context ctx)
