diff --git "a/D:\\Temp\\TortoiseGit\\cr_96A4.tmp\\cr_strings-a306630-left.xml" "b/D:\\Temp\\TortoiseGit\\cr_9694.tmp\\cr_strings-4528e68-right.xml"
index 4434dc0..c9df56b 100644
--- "a/D:\\Temp\\TortoiseGit\\cr_96A4.tmp\\cr_strings-a306630-left.xml"
+++ "b/D:\\Temp\\TortoiseGit\\cr_9694.tmp\\cr_strings-4528e68-right.xml"
@@ -5,7 +5,7 @@
     <!-- About phone screen, temasek version -->
     <string name="temasek_version_title">Temasek unofficial version</string>
     <!-- Do NOT Translate this -->
-    <string name="temasek_version_default">V20.2-private</string>
+    <string name="temasek_version_default">V21</string>
 
     <!-- General strings -->
     <string name="reset">Reset</string>
