diff --git "a/D:\\Temp\\TortoiseGit\\cr_F90.tmp\\cr_strings-a306630-left.xml" "b/D:\\Temp\\TortoiseGit\\cr_F80.tmp\\cr_strings-4528e68-right.xml"
index 0e16305..84eb671 100644
--- "a/D:\\Temp\\TortoiseGit\\cr_F90.tmp\\cr_strings-a306630-left.xml"
+++ "b/D:\\Temp\\TortoiseGit\\cr_F80.tmp\\cr_strings-4528e68-right.xml"
@@ -557,8 +557,12 @@
     <string name="ad_pocket_mode_title">Džepni način</string>
     <string name="ad_pocket_mode_summary">Prikaži obavijesti u aktivnom prikazu kad izvadite uređaj iz džepa.</string>
     <string name="ad_redisplay_title">Ponovni prikaz obavijesti</string>
+    <string name="ad_timeout_title">Istek prikaza</string>
+    <string name="ad_threshold_title">Prag blizine</string>
     <string name="ad_sunlight_mode_title">Način jake svjetlosti</string>
     <string name="ad_sunlight_mode_summary">Obrni boje u područjima jake svjetlosti, poput direktne sunčeve svjetlosti.</string>
+    <string name="ad_turnoff_mode_title">Isključi prikaz</string>
+    <string name="ad_turnoff_mode_summary">Isključi prikaz ako je uređaj vraćen u džep, a bio je upaljen kroz aktivni prikaz.</string>
     <string name="ad_customizations_title">Prilagodbe</string>
     <string name="ad_excluded_apps_title">Isključene aplikacije</string>
     <string name="ad_excluded_apps_summary">Isključene aplikacije neće biti prikazane u aktivnom prikazu.</string>
@@ -576,6 +580,18 @@
     <string name="ad_redisplay_1h">1 sat</string>
     <string name="ad_redisplay_2h">2 sata</string>
     <string name="ad_redisplay_5h">5 sati</string>
+    <string name="ad_timeout_3s">3 sekunde</string>
+    <string name="ad_timeout_5s">5 sekundi</string>
+    <string name="ad_timeout_8s">8 sekundi</string>
+    <string name="ad_timeout_10s">10 sekundi</string>
+    <string name="ad_timeout_15s">15 sekundi</string>
+    <string name="ad_timeout_25s">25 sekundi</string>
+    <string name="ad_threshold_0s">Odmah</string>
+    <string name="ad_threshold_1s">1 sekunda</string>
+    <string name="ad_threshold_2s">2 sekunde</string>
+    <string name="ad_threshold_3s">3 sekunde</string>
+    <string name="ad_threshold_4s">4 sekunde</string>
+    <string name="ad_threshold_5s">5 sekundi</string>
     <string name="ad_pocket_mode_off">Isključeno</string>
     <string name="ad_pocket_mode_notifications">Samo obavijesti</string>
     <string name="ad_pocket_mode_always">Uvijek</string>
@@ -662,6 +678,11 @@
     <string name="title_tile_expanded_desktop">Prošireni prikaz</string>
     <string name="title_tile_music">Glazba</string>
     <string name="title_tile_reboot">Ponovno pokretanje</string>
+    <string name="dynamic_tile_ime_switcher">IME promjena</string>
+    <string name="dynamic_tile_usb_tether">USB povezivanje</string>
+    <string name="dynamic_tile_alarm">Alarm</string>
+    <string name="dynamic_tile_bugreport">Prijava kvara</string>
+    <string name="dynamic_tiles_title">Dinamične pločice</string>
     <string name="tiles_reset_message">Vrati početne postavke?</string>
     <string name="disable_qs">Brze postavke su onemogućene</string>
     <string name="disable_qs_message">Sve pločice su izbrisane. Brze postavke su onemogućene.\n\nDa bi omogućili brze postavke dodajte pločicu.</string>
@@ -672,7 +693,7 @@
     <string name="summary_quicksettings_tiles_style">Pregledajte ili promjenite izgled brzih postavki</string>
     <string name="quicksettings_tiles_style_additional_options">Dodatne mogućnosti</string>
     <string name="tiles_per_row_title">Pločica u redu</string>
-    <string name="tiles_per_row_duplicate_landscape_title">Scale landscape quantity</string>
+    <string name="tiles_per_row_duplicate_landscape_title">Rastegni brojnost u pejsažnom načinu</string>
     <string name="tiles_per_row_duplicate_landscape_summary">Pejsažni prikaz prikazuje dvostruki broj pločica u redu</string>
     <string name="quick_tiles_bg_color_title">Boja pozadine pločica</string>
     <string name="quick_tiles_bg_pressed_color_title">Boja pozadine pritisnute pločice</string>
