diff --git "a/D:\\Temp\\TortoiseGit\\And6E9C.tmp\\AndroidManifest-a306630-left.xml" "b/D:\\Temp\\TortoiseGit\\And6E9B.tmp\\AndroidManifest-4528e68-right.xml"
index 7f47c81..f01c3f6 100644
--- "a/D:\\Temp\\TortoiseGit\\And6E9C.tmp\\AndroidManifest-a306630-left.xml"
+++ "b/D:\\Temp\\TortoiseGit\\And6E9B.tmp\\AndroidManifest-4528e68-right.xml"
@@ -1997,5 +1997,13 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/file_paths" />
         </provider>
+
+        <!-- Lto Service -->
+        <service android:label="LtoService"
+            android:enabled="true"
+            android:exported="false"
+            android:name=".cyanogenmod.LtoService">
+        </service>
+
     </application>
 </manifest>
